.github/workflows/ci.yml

```diff
@@ -318,12 +318,14 @@ jobs:
       - name: build
         run: yarn run build
 
-      - name: build:types
-        run: yarn lerna run build:types
-
       - name: pack
         # Cannot use yarn lerna run pack
         run: yarn lerna exec yarn pack
 
-      - name: clean:types
-        run: yarn lerna run clean:types
+      # Prepack (without cleanup per package) to ensure that type resolution in
+      # dependent packages continues to work when the typedefs are generated by
+      # their upstream packages. This helps avoid a situation in which the types
+      # only resolve because of the state of the local filesystem, and fails
+      # when imported in an NPM node_modules tree.
+      - name: Prepack packages
+        run: yarn lerna run --reject-cycles --concurrency 1 prepack
```

Comment:
I don't understand what the purpose is to generate types after the build is created?

Reply:
Each package is built with access to the entire monorepo source and without other typedefs. When packages are imported, they'll have only the package dependencies and those will have typedefs instead of original src. I've seen errors in resolving at that point.